app/views/components/input_field.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui._
+@import controllers.routes._
+@import models.ocelot.{Labels, Label}
+
+@(input: models.ui.Input, name: String, form: Form[_])(implicit request: Request[_], messages: Messages, labels: Labels)
+
+@optionalAria() = @{
+  Seq(input.hint.map(_ => s"${name}-item-hint"), input.errorMsgs.headOption.map(err => s"${err.id}-error")).flatten match {
+    case Nil => ""
+    case l => ariaDescribedBy(l.mkString(" "))
+  }
+}
+<div class="govuk-form-group">
```

Comment:
There is an HMRC input frontend currency input component we could use here

Reply:
However, it requires some text input whilst render_text generates Html. I guess this is why the components are written as they are.